GrowthViz-adults.py

```diff
@@ -116,23 +116,38 @@
 
 # Next, the `charts.setup_individual_obs_df` function performs transformations on the `cleaned_obs` DataFrame. This will create an `age` column, which is a decimal column that represents the patient's age in years at the time of the observation. It changes the `clean_value` column into a [pandas categorical type](https://pandas.pydata.org/pandas-docs/stable/user_guide/categorical.html). It also create an `include` column which contains a boolean value indicating whether growthcleanr states to include (true) or exclude (false) the observation. The resulting DataFrame is assigned to `obs`.
 
-# In[36]:
+# In[9]:
 
 
 obs_full = charts.setup_individual_obs_df(cleaned_obs)
 
 
 # In the following cell, the `charts.keep_age_range` function visually displays the range of ages in the dataset, with those to be excluded identified by the red bars. It then removes patients outside the intended target population of this notebook (adults 20 to 65).
 
-# In[43]:
+# In[10]:
 
 
 obs = charts.keep_age_range(obs_full)
 
 
+# After that, `charts.weight_distr` creates a visualization to see whether there are spikes at a certain *Included* weights that might indicate that a commonly used scale maxes out at a certain value. The chart is restricted to values of 120kg or higher (rounded to the nearest KG) to make patterns in higher weights easier to identify. This potential issue is important to keep in mind when conducting an analysis.
+
+# In[11]:
+
+
+#charts.weight_distr(obs)
+check = obs[(obs['param'] == 'WEIGHTKG') & (obs['measurement'] >= 120) & (obs['include'] == True)]
+round_col = check.apply(lambda row: np.around(row.measurement, decimals=0), axis=1)
+check = check.assign(round_weight=round_col.values)
+check = check.groupby('round_weight')['subjid'].count().reset_index()
+plt.rcParams['figure.figsize'] = [6, 8]
+check_plot = plt.barh(check['round_weight'], check['subjid']) #, legend=None)
+plt.grid()
```

Comment:
I really like this addition!

I am wondering what it would look like rotated (weights on x-axis, counts on y-axis), and it should probably have axis legends just for clarity.